src/components/CohortBuilder/index.js

```diff
@@ -50,6 +56,50 @@ const CohortBuilder = () => (
       const categoriesSqonUpdate = newSqon => {
         mergeSqonToActiveIndex(newSqon);
       };
+
+      const createNewSqonExcludingParticipants = participantIds => {
+        saveSet({
+          type: 'participant',
+          sqon: {
+            op: 'in',
+            content: {
+              field: 'kf_id',
+              value: participantIds,
+            },
+          },
+          userId: loggedInUser.egoId,
+          path: 'kf_id',
+          api: graphql(api),
+        })
+          .then(({ data }) => {
+            const newSqon = {
+              op: 'and',
+              content: [
+                activeSqonIndex,
+                {
+                  op: 'not',
+                  content: [
+                    {
+                      op: 'in',
```

Comment:
I remember having some weird results with `not-in`, though it might be the sqon using the `set`s that was faulty at that time.
I'll give it another shot.

Reply:
If weird results are coming back, it's likely a back-end issue. We can move forward with `not-in` for the UI and investigate the back-end in a separate task :)